app/assets/javascripts/ama_layout/confirm_modal.js

```diff
@@ -29,13 +29,19 @@ $.rails.confirmed = function(link){
   Toggle the Foundation reveal modal when a data-confirm
   element is clicked.
 */
-$.rails.showConfirmationDialog = function(link){
+$.rails.showConfirmationDialog = function(link) {
   var el = link.data('confirm');
-  $('[data-' + el + ']').foundation('open');
+  var modal = $('[data-' + el + ']');
+  modal.foundation('open');
```

Comment:
Does it make more sense to `open` after setting the data?

Reply:
Sure we could do that - good suggestion!